developers/weaviate/search/multi-vector.md

```diff
@@ -98,6 +101,57 @@ Specify query vectors as a dictionary/map of names and vectors.
 
 You can also specify the query vectors as an array of vectors. The array will be parsed according to the order of the specified target vectors.
 
+### Specify array(s) of query vectors
+
+:::info Added in `v1.27`
+:::
+
+You can specify the value of the query vector dictionary/map as an array of vectors.
```

Comment:
I think this needs a bit more explanation.

How about: 
"You can query the same target vector with multiple query vectors at the same time, by providing a list of vectors for the given target"

Reply:
TY. I agree and have re-written larger portions of this page.